src/Tasks.UnitTests/Microsoft.Build.Tasks.UnitTests.csproj

```diff
@@ -25,7 +25,7 @@
     <ProjectReference Include="..\Tasks\Microsoft.Build.Tasks.csproj" />
     <ProjectReference Include="..\Utilities\Microsoft.Build.Utilities.csproj" />
     <ProjectReference Include="..\Xunit.NetCore.Extensions\Xunit.NetCore.Extensions.csproj" />
-    <ProjectReference Include="..\Samples\PortableTask\PortableTask.csproj" ReferenceOutputAssembly="false" Private="false" SetTargetFramework="TargetFramework=netstandard2.0" />
+    <ProjectReference Include="..\Samples\PortableTask\PortableTask.csproj" ReferenceOutputAssembly="false" Private="false" />
```

Comment:
no need to pass SetPlatform netstandard2.0 because PortableTask now only builds as 2.0.

What's weird is that PortableTask used to have TargetFramework**s** ns2.0 instead of `TargetFramework`.

Reply:
Before that, it used to multitarget 🤷🏻‍♂️